docker/README.md

```diff
@@ -1,239 +1,269 @@
-- [Basic Usage](#usage)
-- [Development](#development)
 
-# Overview
+## BentoServer base images
 
-There are three type of BentoServer base image:
+There are four type of BentoServer docker base image:
 
 | Image Type | Description                                | Supported OS                                          | Usage                             |
 |------------|--------------------------------------------|-------------------------------------------------------|-----------------------------------|
-| `runtime`  | contains latest BentoML releases from PyPI | `debian`, `ubi{7,8}`, `amazonlinux2`, `alpine3.14` | production ready                  |
-| `cudnn`    | runtime + support for CUDA-enabled GPU     | `debian`, `ubi{7,8}`                               | production ready with GPU support |
-| `devel`    | nightly build from development branch      | `debian`, `ubi{7,8}`                               | for development use only          |
+| `runtime`  | contains latest BentoML releases from PyPI | `debian{11,10}`, `ubi8`, `amazonlinux2`, `alpine3.14` | production ready                  |
+| `cudnn`    | runtime + support for CUDA-enabled GPU     | `debian{11,10}`, `ubi8`                               | production ready with GPU support |
+| `devel`    | nightly build from development branch      | `debian{11,10}`, `ubi8`                               | for development use only          |
+| `conda`    | runtime + conda + optional GPU supports    | `debian{11,10}`,                                      | production ready                  |
 
 * Note: currently there's no nightly devel image with GPU support.
 
 The final docker image tags will have the following format:
 
 ```markdown
-<release_type>-<python_version>-<distros>-<suffix>
+<release_type>-<python_version>-<distros>-<suffix>-<?:conda>
    │             │                │        │
    │             │                │        └─> additional suffix, differentiate runtime and cudnn releases
    │             │                └─> formatted <dist><dist_version>, e.g: ami2, debian, ubi7
    │             └─> Supported Python version: python3.7 | python3.8 | python3.9
-   └─>  Release type: devel or official BentoML release (e.g: 1.0.0)                                           
+   └─>  Release type: devel or official BentoML release (e.g: 1.0.0)
 ```
 
 Example image tags:
 - `bento-server:devel-python3.7-debian`
 - `bento-server:1.0.0-python3.8-ubi8-cudnn`
 - `bento-server:1.0.0-python3.7-ami2-runtime`
 
-See all available tags [here](https://hub.docker.com/repository/docker/bentoml/bento-server/tags).
+## NOTICE: MISSING PYTHON VERSION ON UBI
 
+Python 3.7 and 3.10 is missing. The reason being RedHat doesn't provide support for these Python version.
+If you need to use UBI and Python 3.7 make sure to contact the BentoML team for supports..
+
+## NOTICE: CONDA AVAILABILITY ONLY ON DEBIAN
+
+From 1.0.0a7 onwards, BentoML will only provide conda supports with debian variants only.
```

Comment:
add a check for this in BentoBuildConfig, that verify CondaOptions.is_empty and DockerOptions.distro are compatible